CHANGELOG.md

```
@@ -2,13 +2,36 @@
 
-The current Android release candidate is **0.7.6** (`versionCode 26`).
+The current Android release candidate is **0.7.7** (`versionCode 27`).
 The Web/PWA version remains **0.7.3** and is versioned independently.
 
 The format is loosely based on [Keep a Changelog](https://keepachangelog.com/).
 
 ---
 
+## [0.7.7] — safer sharing, Search Assistant risk parity, local rating prompt
+
+### Added
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)